_posts/config/2017-05-06-wi-fi-configuration.md

```diff
@@ -0,0 +1,206 @@
+
+- **Password Verification** - clicking this button triggers password verification before a full connection is established.
+
+## Wi-Fi Linux Configuration
+
+This section describes the changes applied by Kura at the Linux networking configuration. Please read the following note before proceeding with manual changes of the Linux networking configuration.
+
+{% include alerts.html message='It is NOT recommended performing manual editing of the Linux networking configuration files when the gateway configuration is being managed through Kura. While Linux may correctly accept manual changes, Kura may not be able to interpret the new configuration resulting in an inconsistent state.' %}
+
+When the Wi-Fi configuration for the _Access Point_ mode is submitted, Kura generates the /etc/hostapd.conf file and launches the hostapd program as shown below.
+
+```
+hostapd:B /etc/hostapd.conf
+```
+
+```
+# /etc/hostapd/hostapd.conf
+
+interface=wlan0
+
+driver=nl80211
+
+# SSID to use. This will be the "name" of the accesspoint
+ssid=kura_gateway_00:E0:C7:09:35:D8
+
+# basic operational settings
+hw_mode=g
+
+wme_enabled=0
+ieee80211n=0
+
+channel=1
+
+# Logging and debugging settings: more of this in original config file
+logger_syslog=-1
+logger_syslog_level=2
+logger_stdout=-1
+logger_stdout_level=2
+dump_file=/tmp/hostapd.dump
+
+# WPA settings. We'll use stronger WPA2
+# bit0 = WPA
+# bit1 = IEEE 802.11i/RSN (WPA2) (dot11RSNAEnabled)
+wpa=2
+
+# Preshared key of between 8-63 ASCII characters.
+# If you define the key in here, make sure that the file is not readable
+# by anyone but root. Alternatively you can use a separate file for the
+# key; see original hostapd.conf for more information.
+wpa_passphrase=testKEYS
+
+# Key management algorithm. In this case, a simple pre-shared key (PSK)
+wpa_key_mgmt=WPA-PSK
+
+# The cipher suite to use. We want to use stronger CCMP cipher.
+wpa_pairwise=CCMP
+
+# Change the broadcasted/multicasted keys after this many seconds.
+wpa_group_rekey=600
+
+# Change the master key after this many seconds. Master key is used as a basis
+# (source) for the encryption keys.
+wpa_gmk_rekey=86400
+
+# Send empty SSID in beacons and ignore probe request frames that do not
+# specify full SSID, i.e., require stations to know SSID.
+# default: disabled (0)
+# 1 = send empty (length=0) SSID in beacon and ignore probe request for
+#     broadcast SSID
+# 2 = clear SSID (ASCII 0), but keep the original length (this may be required
+#     with some clients that do not support empty SSID) and ignore probe
+#     requests for broadcast SSID
+ignore_broadcast_ssid=0
+```
+
+When the Wi-Fi configuration for the _Access Point_ mode is submitted, Kura generates the /etc/wpa_supplicnt.conf file and launches the wpa_supplicant program as shown below.
+
+```
+wpa_supplicant:B:D nl80211:i wlan0:c /etc/wpa_supplicant.conf
+```
```

Comment:
What I get is just a bit different:
wpa_supplicant -B -D nl80211 -i wlan0 -c /etc/wpa_supplicant-wlan0.conf